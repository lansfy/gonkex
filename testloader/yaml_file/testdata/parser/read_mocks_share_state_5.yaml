- name: WHEN options shareState starts without mocks definition parser MUST fail
  method: GET
  path: /some/path1
  request: "requestbody"
  mocksParams:
    shareState: true
