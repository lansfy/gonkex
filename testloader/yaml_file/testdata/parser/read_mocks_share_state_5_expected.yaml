- Error: "process 'testdata/parser/read_mocks_share_state_5.yaml': test 'WHEN options shareState starts without mocks definition parser MUST fail': shareState require non empty $.mocks declaration"
